src/hermes/model/context.py

```diff
@@ -119,6 +120,14 @@ def error(self, ep: EntryPoint, error: Exception):
 
         self._errors.append((ep, error))
 
+    def purge_caches(self, *path: str) -> Path:
```

Comment:
```suggestion
    def purge_caches(self, *path: List[str]) -> None:
```

This method doesn't return anything. Also `path` should probably be a list of str or a list of pathlib.Path.

The path parameter is unused. If it doesn't make sense to implement this now but is supposed to be implemented later, maybe add:

```python
if path:
    raise NotImplementedError()
```

Reply:
OK, the List thing is probably wrong, sorry 😄

Reply:
The `path` argument is obsolete, because the path of the hermes project is provided by the context. I first thought, I need to pass the path to the function an the forgot to remove it once I figured-out the context.